src/components/searchBar/SearchBar.tsx

```diff
@@ -0,0 +1,52 @@
+import { InputAdornment, styled } from '@mui/material'
+import { useState } from 'react'
+
+import { Search } from '@mui/icons-material'
+import { StyledTextField } from '../inputs/TextField'
+
+interface ISearchBar {
+  value: string
+  getSearchKeyword: (keyword: string) => void
+}
+
+const SearchBar = ({ value, getSearchKeyword }: ISearchBar) => {
+  const [focused, setFocused] = useState(false)
+
+  return (
+    <StyledTextInput
+      value={value}
+      focused={focused}
+      variant="outlined"
+      placeholder={focused ? 'Search...' : 'Search'}
+      size="small"
+      InputProps={{
+        startAdornment: (
+          <InputAdornment
+            position="start"
+            sx={{
+              cursor: 'pointer',
+            }}
```

Comment:
The cursor for a search field makes more sense to be a text beam than a pointer
```suggestion
            position="start"
            sx={{
              cursor: 'text',
            }}
```

Reply:
@rrojan @arpandhakal 
This should be `pointer` itself which is accordance to the Search field component being used all over the Copilot dashboard [see here for example](https://dashboard.copilot.com/clients/users). This `SearchBar` component is the copy-paste of the same code that I wrote for the Profile Manager app.

Reply:
@aatbip This makes sense... it does look a bit weird though

cc @foleyatwork